src/VisualStudio/VisualStudioDiagnosticsToolWindow/OptionPages/PerformanceLoggersPage.cs

```diff
@@ -35,17 +35,13 @@ protected override void OnApply(PageApplyEventArgs e)
 
             if (_optionService.GetOption(LoggerOptions.EtwLoggerKey))
             {
-                Logger.SetLogger(AggregateLogger.AddOrReplace(new EtwLogger(_optionService), Logger.GetLogger(), l => l is EtwLogger || l is TraceLogger));
-                return;
+                Logger.SetLogger(AggregateLogger.AddOrReplace(new EtwLogger(_optionService), Logger.GetLogger(), l => l is EtwLogger));
```

Comment:
this was not 100% related to this change, but needed to make investigation easier.